docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -134,6 +134,11 @@ SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguratio
 let lockScreenViewController = <# Initialize Your View Controller #>
 let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(with: lockScreenViewController)
 ```
+
+!!! NOTE
```

Comment:
Remove the `!!! NOTE ... !!!` formatting tags and just make the new text a paragraph. Instead of having a separate paragraph, just add it after the sentence above (_If you would like to use your own lock screen .._.).